core/mpc/keygenComplete.ts

```
@@ -0,0 +1,69 @@
+import { retry } from "@lib/utils/query/retry"
```

ESLint

[error] Replace `"@lib/utils/query/retry"` with `'@lib/utils/query/retry'`

(prettier/prettier)